homeassistant/components/ituran/coordinator.py

```diff
@@ -0,0 +1,67 @@
+"""Coordinator for Ituran."""
+
+from datetime import timedelta
+import logging
+
+from pyituran import Ituran, Vehicle
+from pyituran.exceptions import IturanApiError, IturanAuthError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import CONF_ID_OR_PASSPORT, CONF_MOBILE_ID, CONF_PHONE_NUMBER, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class IturanDataUpdateCoordinator(DataUpdateCoordinator[dict[str, Vehicle]]):
+    """Class to manage fetching Ituran data."""
+
+    ituran: Ituran
```

Comment:
```suggestion
    ituran: Ituran
    config_entry: IturanConfigEntry
```
you need some typing though